pkg/descriptions.json

```diff
@@ -1398,6 +1398,16 @@
     ]
   },
   {
+    "Name": "System.Runtime.Intrinsics.X86",
+    "Description": "Provides classes which expose low-level processor intrinsics for the x86 family of instruction set architectures.",
+    "CommonTypes": [
+      "System.Runtime.Intrinsics.X86.Sse",
+      "System.Runtime.Intrinsics.X86.Avx",
+      "System.Runtime.Intrinsics.X86.Popcnt",
+      "System.Runtime.Intrinsics.X86.Fma",
```

Comment:
Don't we just need these 4 classes?

Reply:
This is just a list of "commonly-used types", not all of them. I think it is just used to help people search for the library, so I tried to pick a few of the most-common ones that people will try to find.

Does that make sense? I may have misunderstood your question.

Reply:
> Does that make sense? 

Yes, thanks. I am not familiar with the Nuget package system, so just a quick question.